app/views/facility_accounts/account_fields/_nufs_account.html.haml

```diff
@@ -1 +1,6 @@
+- is_new ||= false
```

Comment:
I was hoping to default to false and only use `true` if it's passed in.  I can use `local_assigns` if you like but I think it's more code to achieve the same thing.  Although I'm all for it if you believe it's easier to understand what's going on.